papers/unlocking_78/myst.yml

```
@@ -0,0 +1,50 @@
+version: 1
+extends: ../papers.yml
+project:
+  id: scipy-2025-missing_78
+  title: "Unlocking the Missing 78%: Inclusive Communities for the Future of Scientific Python"
+  description: This article quantifies the gender gap in the scientific Python community, analyzes its root causes, and proposes the Visibility–Invitation–Mechanism (VIM) framework as a practical solution. A case study of the IBM Women in AI (WAI) User Group is presented to demonstrate the framework's effectiveness.
+  authors:
+    - name: Noor Aftab
+      email: noor.aftab@gmail.com
+      affiliations:
+        - IBM Champion, PyData Speaker, Chair of NumFOCUS CoC WG
+      roles:
+      - Conceptualization
+      - Investigation
+      - Methodology
+      - Software
+      - Validation
+      - Visualization
+      - Writing – Original Draft
+      - Writing – Review & Editing
+      corresponding: true
+  keywords:
+    - scientific python
+    - diversity
+    - inclusion
+    - gender gap
+    - open source
+    - community
+    - VIM framework
+  abbreviations:
+    VIM: Visibility–Invitation–Mechanism
+    WAI: Women in AI
+    CoP: Communities of Practice
+    OSS: Open-Source Software
+    DEI: Diversity, Equity, and Inclusion
+  error_rules:
+    - rule: doi-exists
+      severity: ignore
+      keys:
+        - Atr03
+        - terradesert
+        - jupyter
+        - sklearn1
+        - sklearn2
+  exports:
+    - id: pdf
+      format: typst
+      template: https://github.com/curvenote-templates/scipy.git
+      article: unlocking_78.md
+      output: unlocking_78.pdf
```

For now, this output must be named like so for the PDF to show up in your preview build:

```suggestion
      output: full_text.pdf
```